schemas/triniti/ovp.medialive/enums.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-enums xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/enums.xsd">
+
+  <enums namespace="triniti:ovp.medialive">
+    <!--
+      @link https://docs.aws.amazon.com/medialive/latest/apireference/channels.html#channels-properties
+    -->
+    <enum name="medialive-channel-status" type="string">
```

Comment:
@efjacobson isn't it media live channel state?

Reply:
@gdbrown done